src/main/webapp/WEB-INF/views/drivers/drivers.jsp

```diff
@@ -0,0 +1,34 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Drivers</title>
+</head>
+<body>
+<h1>Drivers</h1>
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>License number</th>
+    </tr>
+    <c:forEach var="driver" items="${drivers}">
+        <tr>
+            <td>
+                <c:out value="${driver.id}"/>
+            </td>
+            <td>
+                <c:out value="${driver.name}"/>
+            </td>
+            <td>
+                <c:out value="${driver.licenseNumber}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/drivers/delete?id=${driver.id}">Delete</a>
+            </td>
```

Comment:
do you have this implemented?

Reply:
taken it from Bohdan's videolesson, but not implemented. Lefted for future implementation if it's going to be needed in dao9.